src/styles/index.scss

```diff
@@ -1,3 +1,135 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+article {
+  display: inline-block;
+}
+
+.main-container {
+  width: 166px;
+  padding: 32px 15px 16px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  border: 1px solid #fff;
+  border-radius: 5px;
+}
+
+.main-container__product-img {
+  max-height: 134px;
+  max-width: 160px;
+  display: flex;
+  margin-bottom: 40px;
+}
+
+.main-container__description {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  justify-content: space-evenly;
+}
+
+.main-container__rating-of-product {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  margin-bottom: 17px;
+}
+
+.main-container__product-price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  height: min-content;
+  margin-top: 6px;
+  margin-bottom: 14px;
+}
+
+.main-container__stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.main-container__stars--4 :nth-child(-n + 1),
+.main-container__stars--4 :nth-child(-n + 2),
+.main-container__stars--4 :nth-child(-n + 3),
+.main-container__stars--4 :nth-child(-n + 4),
+.main-container__stars--4 :nth-child(-n + 5) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.main-container__stars {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.main-container__button-buy {
+  background-color: rgba(0, 172, 220, 1);
+  border-radius: 5px;
+  border: 1px solid rgba(0, 172, 220, 1);
+}
+
+.main-container__button-buy:hover {
+  background-color: #fff;
+  border: 1px solid rgba(0, 172, 220, 1);
+}
+
+.main-container__button-link {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  color: #fff;
```

Comment:
Create file variables, for all colors, and use them inside all SCSS files.
Thanks